src/services/subsocial/hooks.ts

```diff
@@ -38,7 +62,11 @@ export default function useCommonTxSteps<Data, ReturnValue>(
     }
 
     if (!hasEnoughEnergy && captchaToken) {
-      await requestToken({ address: usedAddress, captchaToken })
+      const [_, res] = await Promise.all([
+        requestToken({ address: usedAddress, captchaToken }),
+        mutateAsync(params),
```

Comment:
Is it ok, what if energy dont recived yet?

Reply:
right now it's also waiting for energy in the mutateAsync

The issue why this is needed is because, the onStart (the callback where the tx starts, where I usually set my client optimistic updates) will only be called once the mutateAsync is called
if I await requestToken first, it will just add additional time for the user to see the optimistic updates, while the mutation itself will still need to wait for the energy to arrive at that account